lib/cocoapods/sandbox/file_accessor.rb

```diff
@@ -305,17 +301,31 @@ def docs
         path_list.glob([GLOB_PATTERNS[:docs]])
       end
 
+      # @return [Pathname] The path of the license file specified in the
+      #         specification, if it exists
+      #
+      def spec_license
+        if file = spec_consumer.license[:file]
+          absolute_path = root + file
+          absolute_path if File.exist?(absolute_path)
+        end
+      end
+
       # @return [Array<Pathname>] Paths to include for local pods to assist in development
       #
       def developer_files
         podspecs = specs
         result = [module_map, prefix_header]
+        validate_spec_license
```

Comment:
nit could also inline this method as there is no other callers for it

Reply:
Good point - will inline it